src/core/packages/rendering/server-internal/src/bootstrap/bootstrap_renderer.test.ts

```diff
@@ -290,13 +290,17 @@ describe('bootstrapRenderer', () => {
 
       expect(getThemeTagMock).toHaveBeenCalledTimes(1);
       expect(getThemeTagMock).toHaveBeenCalledWith({
-        name: 'v8',
+        name: 'borealis',
         darkMode: true,
       });
     });
 
     it('calls getThemeTag with the correct parameters when darkMode is `system`', async () => {
-      uiSettingsClient.get.mockResolvedValue('system');
+      uiSettingsClient.get.mockImplementation(
```

Comment:
This test was wrongly updated in a prior PR. `system` is the value returned for `theme:darkMode` and is not a valid `theme:name` value.